### DISCUSSION TITLE
Qns_about / help_with; YTDLnis

### DISCUSSION THREAD

**Discussion by J-HS-L** [deniscerri/ytdlnis#671] [Q&A]
Is it possible to use this app to export my many yt playlists, to xyz format(s)? 

Also, where do I log into my YouTube account? 

I can't find that, so that I may download entire (or portions) playlists! 

Galaxy S23 Ultra, latest stable Android & OneUI. 

Thank you 🙏🤗

**Comment by error-reporting** [comment]
export playlist to video or csv?

**Comment by error-reporting** [comment]
yes
login here
